apps/api/v2/src/ee/bookings/2024-04-15/controllers/bookings.controller.ts

```diff
@@ -238,7 +242,7 @@ export class BookingsController {
   @Post("/recurring")
   async createRecurringBooking(
     @Req() req: BookingRequest,
-    @Body() _: CreateRecurringBookingInput[],
+    @Body() _: CreateRecurringBookingInput_2024_04_15[],
     @Headers(X_CAL_CLIENT_ID) clientId?: string
   ): Promise<ApiResponse<BookingResponse[]>> {
     const oAuthClientId = clientId?.toString();
```

Comment:
Is this compatible with API requests as well, or just OAuthClient?

Reply:
Compatible with both